usaspending_api/references/constants.py

```diff
@@ -27,3 +27,6 @@
     ("011", "8238"),
     ("011", "8242"),
 ]  # Federal Account (AID, MAIN)s that are to be reported under DoD.
+
+# Agencies which should be excluded from dropdowns.
+EXCLUDE_CGAC = ["000", "067"]
```

Comment:
Why was this added back?  It's obsolete now.

Reply:
Must have been an artifact of reverting stuff when I was adding constants in the old implementation.